generated/meson.build

```diff
@@ -0,0 +1,55 @@
+
+sourcery = executable('sourcery', join_paths(meson.source_root(), 'src', 'sourcery.cpp'), native: true)
+
+make_html = executable('make_html', 'make_html.cpp', native: true)
+
+html_list = [
+    join_paths(meson.source_root(), 'lsp/header.html'),
+    join_paths(meson.source_root(), 'lsp/main.css'),
+    join_paths(meson.source_root(), 'lsp/footer.html'),
+    join_paths(meson.source_root(), 'lsp/plugins/jquery.js'),
+    join_paths(meson.source_root(), 'lsp/plugins/jquery.flot.min.js'),
+    join_paths(meson.source_root(), 'lsp/plugins/jquery.flot.time.min.js'),
+    join_paths(meson.source_root(), 'lsp/plugins/jquery.qrcode.min.js'),
+    join_paths(meson.source_root(), 'lsp/minified.js'),
+]
+
+if not get_option('NOGA')
+    html_list += join_paths(meson.source_root(), 'lsp/analytics.js')
+endif
+
+gen_html = custom_target('gen_html', output: 'server.html', input: html_list, command: [make_html, '@INPUT@'], capture: true)
```

Comment:
Because make_html is controlled by you and is even created in this commit...

I would suggest having it accept the output filename as another command line argument, and writing directly to that instead of using `capture: true`.

This means Meson doesn't have to invoke `meson --internal exe --capture=@OUTPUT@ path/to/make_html @INPUT@` which is a python wrapper script that just runs make_html, pipes the stdout into a buffer, and then writes the buffer to the output file for you.

The python overhead can be a teensy bit annoying, and it makes it less straightforward to see the actual command being run.